src/tests/Common/scripts/nativeaottest.sh

```diff
@@ -10,6 +10,13 @@
 # 2. Filename of the test executable
 # 3. - n. Additional arguments that were passed to the test .sh
 
-exename=$(basename $2 .dll)
-chmod +x $1/native/$exename
-$_DebuggerFullPath $1/native/$exename "${@:3}"
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+
+source $SCRIPT_DIR/../../../../eng/testing/FindWasmHostExecutable.sh "$1/native/$2"
+
+if [ -n "${WASM_HOST_EXECUTABLE}" ]; then
+  $WASM_HOST_EXECUTABLE "$WASM_BINARY_TO_EXECUTE" "${@:3}"
+else
+  chmod +x $1/native/$exename
+  $_DebuggerFullPath $1/native/$exename "${@:3}"
```

Comment:
What sets `exename` now?

Reply:
reverted the deleting of setting `exename`